lib/trie/database.go

```diff
@@ -432,14 +444,14 @@ func (t *Trie) writeDirty(db chaindb.Batch, n Node) error {
 // We need to compute the hash values of each newly inserted node.
 func (t *Trie) GetInsertedNodeHashes() (hashesSet map[common.Hash]struct{}, err error) {
 	hashesSet = make(map[common.Hash]struct{})
-	err = t.getInsertedNodeHashes(t.root, hashesSet)
+	err = t._getInsertedNodeHashes(t.root, hashesSet)
```

Comment:
Why the name change? Please revert

Reply:
Commented below on RVV-B0001 linter issue.